src/config.ts

```diff
@@ -22,7 +22,17 @@ type BaseConfig = {
    * URL of the Storybook instance or local folder
    * @default 'storybook-static'
    */
-  storybookUrl: string;
+  storybookUrl?: string;
```

Comment:
I struggled to make typescript work - maybe you could give a hand @chriskalmar :/ We need either `storybookUrl` or union of `pages`&`pageBaselineUrl`, both can't be present in the config currently